### ISSUE TITLE
got NAN after trainging some iters

### ISSUE THREAD

**Issue by harlem867** [motokimura/PyTorch_Gaussian_YOLOv3#12]
@motokimura dear motokimura,
when I use your code to train COCＯ dataset, the loss declined normally and suddenly xy and wh loss changed to nan after about 10w iters. The learning rate is 0.001 during the whole training process and i didn't change your default settings.
When I trained on my own dataset, got the same problem.
Can you give me some advice? Thank you.

**Comment by motokimura** [comment]
Hi, can you try with smaller learning rate?
Making `GRADIENT_CLIP` smaller might help also.

https://github.com/motokimura/PyTorch_Gaussian_YOLOv3/blob/b51eb917ede2a1d7bc6eb0e220bf330f43baf842/config/gaussian_yolov3_default.cfg#L22

**Comment by motokimura** [comment]
Or, it might help to add constant offsets to predicted uncertainties as done in official implementation. For details, please see https://github.com/jwchoi384/Gaussian_YOLOv3/issues/40.
